Onboarding/re-engagement.md

```diff
@@ -0,0 +1,431 @@
+  ) do
+    text("@loading_message")
+  end
+end
+
+# Show image
+card DisplayReminderRequest, then: ReminderRequestError do
+  image_id = page.body.body.text.value.image
+
+  image_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/images/@image_id/",
+      headers: [
+        ["Authorization", "Token @global.config.contentrepo_token"]
+      ]
+    )
+
+  buttons(
+    SaveReengagement: "@button_labels[0]",
+    RemindMe: "@button_labels[1]"
+  ) do
+    image("@image_data.body.meta.download_url")
+    text("@loading_message")
+  end
+end
+
+card ReminderRequestError, then: ReminderRequestError do
+  buttons(
+    SaveReengagement: "@button_labels[0]",
+    RemindMe: "@button_labels[1]"
+  ) do
+    text("@button_error_text")
+  end
+end
+
+```
+
+## Go To Where User Dropped Off
+
+```stack
+card SaveReengagement, then: DropOff1stReminder do
+  write_result("reengaged_point", contact.reengagement_message)
+end
+
+card DropOffGoTo when contact.checkpoint == "generic_basic_info", then: BasicProfile do
+  write_result("reengagement_point", "30%")
+  log("Go to BasicProfile")
+end
+
+card DropOffGoTo when contact.checkpoint == "generic_personal_info", then: PersonalProfile do
+  write_result("profile_completion", "30%")
+  log("Go to PersonalProfile")
+end
+
+card DropOffGoTo when contact.checkpoint == "generic_daily_life_info", then: LOCAssessment do
+  write_result("profile_completion", "30%")
+  log("Go to LOCAssessment")
+end
+
+card DropOffGoTo when contact.checkpoint == "pregnant_nurse_profile", then: NurseQuestions do
+  write_result("profile_completion", "30%")
+  log("Go to NurseQuestions")
+end
+
+card DropOffGoTo, then: PregnancyQuestions do
+  write_result("profile_completion", "30%")
+  log("Go to PregnancyQuestions")
+end
+
+```
+
+## Remind me tomorrow
+
+Will resend a reminder after 23 hours
+
+```stack
+card RemindMe do
+  update_contact(reengagement_message: "remind me")
+  schedule_stack("b11c7c9c-7f02-42c1-9f54-785f7ac5ef0d", in: 23 * 60 * 60)
+end
+
+```
+
+## Pregnancy Questions
+
+```stack
+card PregnancyQuestions do
+  log("Pregnancy Questions")
+  run_stack("d5f5cfef-1961-4459-a9fe-205a1cabfdfb")
+end
+
+```
+
+## Nurse Questions
+
+```stack
+card NurseQuestions do
+  log("Nurse Questions")
+  run_stack("38cca9df-21a1-4edc-9c13-5724904ca3c3")
+end
+
+```
+
+## Basic Profile
+
+```stack
+card BasicProfile do
+  log("Basic Profile")
+  run_stack("26e0c9e4-6547-4e3f-b9f4-e37c11962b6d")
+end
+
+```
+
+## Personal Profile
+
+```stack
+card PersonalProfile do
+  log("Personal Profile")
+  run_stack("61a880e4-cf7b-47c5-a047-60802aaa7975")
+end
+
+```
+
+## LOC Assessment
+
+```stack
+card LOCAssessment do
+  log("Loc Assessment")
+  # run_stack("d5f5cfef-1961-4459-a9fe-205a1cabfdfb")
+end
+
+```
+
+## Schedule for 1 after later
+
+Need to set this at the start od onboarding
+
+```stack
+card Schedule1hLater do
+  schedule_stack("b11c7c9c-7f02-42c1-9f54-785f7ac5ef0d", in: 60 * 60)
+end
```

Comment:
I think this needs to move into Intro & Onboarding right at the top (either just before or just after FetchError).